app/assess/routes.py

```diff
@@ -110,14 +111,21 @@ def display_sub_criteria(
             abort(404)
         # SECURITY SECTION END ######
 
-        form = ScoreForm()
-        score_error = justification_error = scores_submitted = False
+        score_form = ScoreForm()
+        rescore_form = RescoreForm()
+        score_error = (
+            justification_error
+        ) = scores_submitted = is_rescore = False
         if request.method == "POST":
             current_app.logger.info(f"Processing POST to {request.path}.")
-            if form.validate_on_submit():
-                score = int(form.score.data)
+            if rescore_form.validate_on_submit():
```

Comment:
can we add some comments as there are multiple forms?

Reply:
have added doc-strings in form classes and comment in this line also 👍🏾